### ISSUE TITLE
Is it done?

### ISSUE THREAD

**Issue by Tom8thepie** [Bobakanoosh55555/EZSwitchTAS#1]
I’ve been very interested lately in creating tas scripts for my friends who own arduinos and I just want a way to do that easily and this is the only way I could possibly do this

**Comment by Bobakanoosh55555** [comment]
Hey Tom,

Love to see interest in stuff like this! I'm not sure my repo will be of much help... its an attempt to make a TAS tool during my college hackathon in early 2019. I had it working at some point, but during the latter half everything simply stopped working and I was never able to fix it. 

The silver lining is that a majority of this project was built off of / adapted the code of the repo "Switch-Fightstick", formerly known as "Splatoon-Printer" or something of the likes. I've linked the repo below - the code was fairly straightforward to understand, and was very helpful in making basic TAS's or feed any input into the switch through USB. I hope you have better luck than I did!

https://github.com/shinyquagsire23/Switch-Fightstick

If you have any questions, I can do my best to help out!

**Comment by Tom8thepie** [comment]
Thanks! I actually do have one question, how would you modify the code to press multiple inputs on the same frame?

**Comment by Bobakanoosh55555** [comment]
The code should be able to do that by default - the original code base was made for adding fightstick support and the splatoon printer, both of which used multiple inputs on the same frame

**Comment by Tom8thepie** [comment]
Ok thank you, I just had my first success with the switch recognizing the arduino as a controller and I will see where I can take it from there based on the advice you have given me, cheers!

**Comment by Bobakanoosh55555** [comment]
Hey Tom,
Love to see interest in stuff like this! I'm not sure my repo will be of much help... its an attempt to make a TAS tool during my college hackathon in early 2019. I had it working at some point, but during the latter half everything simply stopped working and I was never able to fix it. 
The silver lining is that a majority of this project was built off of / adapted the code of the repo "Switch-Fightstick", formerly known as "Splatoon-Printer" or something of the likes. I've linked the repo below - the code was fairly straightforward to understand, and was very helpful in making basic TAS's or feed any input into the switch through USB. I hope you have better luck than I did!

https://github.com/shinyquagsire23/Switch-Fightstick


    On Thursday, May 12, 2022, 07:30:21 PM EDT, Tom8thepie ***@***.***> wrote:  
 
 


I’ve been very interested lately in creating tas scripts for my friends who own arduinos and I just want a way to do that easily and this is the only way I could possibly do this
